packages/react-ui/src/widgets/legend/LegendWidgetUI.js

```diff
@@ -61,7 +62,7 @@ function LegendWidgetUI({
         collapsed={collapsed}
         onCollapsedChange={onCollapsedChange}
       >
-        <LegendRows layers={layers} onChangeVisibility={onChangeVisibility} />
+        <LegendRows layers={layers} onChangeVisibility={onChangeVisibility} onChangeOpacity={onChangeOpacity} />
```

Comment:
I'm getting here "'LegendRows' cannot be used as a JSX component." warning in VS Code, because it looks like it returns an array, should we just add a fragment there?